Xamarin.Forms.Core/SwipeView.cs

```diff
@@ -95,6 +95,13 @@ public SwipeEndedEventArgs(SwipeDirection swipeDirection) : base(swipeDirection)
 			}
 		}
 
+		SwipeItems SwipeItemsDefaultValueCreator() => new SwipeItems();
+
+		static object SwipeItemsDefaultValueCreator(BindableObject bindable)
+		{
+			return ((SwipeView)bindable).SwipeItemsDefaultValueCreator();
+		}
+
```

Comment:
you can probably inline all that as a single lambda, and copy it in each property